src/conf_mode/interfaces-openvpn.py

```diff
@@ -899,6 +899,13 @@ def generate(openvpn):
     interface = openvpn['intf']
     directory = os.path.dirname(get_config_name(interface))
 
+    # we can't know which clients were deleted, remove all client configs
+    if os.path.isdir(directory + '/ccd/' + openvpn['intf']):
```

Comment:
This works but a better way would be to `os.path.join(directory,'ccd',openvpn['intf'])`

Reply:
Fixed